Resources/Audio/Items/attributions.yml

```diff
@@ -52,3 +52,8 @@
   license: "CC-BY-SA-3.0"
   copyright: "Taken from /tg/station, edited by Flareguy for Space Station 14"
   source: "https://github.com/tgstation/tgstation/blob/6665eec76c98a4f3f89bebcd10b34b47dcc0b8ae/sound/weapons/armbomb.ogg"
+
+- files: ["Toys/meow2.ogg"]
+  license: "CC0-1.0"
+  copyright: "Created by suicdxsaturday, shortened and converted to OGG and mono"
+  source: "https://freesound.org/people/suicdxsaturday/sounds/698633/"
```

Comment:
Was it recorded by you (your cat) or did you just take it from somewhere and uploaded on freesound?

Reply:
as written in the file, only it was free and im downloaded and "shortened and converted to OGG and mono"

Reply:
It isn't said in this file that is was free and you just made it look the sound is yours.

Reply:
That is, what kind of license do I need to specify all the same? I actually changed the sound...

Reply:
Look at other sounds and how they license it, nobody wipes out the OG author of the sound even if you modified it. And answer the most important question about the sprites copyright/license

Reply:
The sprite is freely available on the Internet. But still i tried to redraw it, in NecoBlack i tried to make more changes. But if you think that it does not fit into the game, then this is your opinion.

Reply:
The sprite is freely available on the Internet. But still i tried to redraw it, in NecoBlack i tried to make more changes. But if you think that it does not fit into the game, then this is your opinion.

Reply:
![](https://user-images.githubusercontent.com/46868845/259514660-e539b4ed-bd6b-4f6e-89a6-a1442e554926.png)

Reply:
"Also, there is no problem with the use of newly drawn illustrations based on images of TYPE-MOON works"

The sprite is not from the original "painting", Yes, it is based on the works of TYPE-MOON, I don't really understand, is there a problem?